core/Makefile

```diff
@@ -88,8 +91,8 @@ ifdef COVERAGE
 endif
 
 ifndef DEBUG
-    CXXFLAGS += -Ofast -DNDEBUG
-    LDFLAGS += -ffat-lto-objects -finline-limit=1000
+    CXXFLAGS += -Ofast -DNDEBUG -flto -ffat-lto-objects -finline-limit=1000
+    LDFLAGS += -flto -ffat-lto-objects -finline-limit=1000
```

Comment:
Only add -flto when the compiler is not Clang (as we did in the following lines), so we can remove -flto in this line.